uswds/uswds-next
# USWDS Web-Components - POAM: December ‘24

# Summary

Dependency updates for December 2024

## Breaking change

This is not a breaking change.

## Related issue

[USWDS - POAM: Dec '24](https://github.com/uswds/uswds-team/issues/413)

## Preview link

Preview link →

## Major changes

- Installed available minor and patch changes to dependencies

## Dependency updates

**Before:**

```css
1 low severity vulnerability
```

**After:**

```css
found 0 vulnerabilities
```

| Dependency Name | Old Version | New Version |
| --- | --- | --- |
| @rollup/rollup-linux-x64-gnu | ^4.27.3 | ^4.28.1 |
| @storybook/addon-a11y | ^8.4.4 | ^8.4.7 |
| @storybook/addon-docs | ^8.4.4 | ^8.4.7 |
| @storybook/addon-essentials | ^8.4.4 | ^8.4.7 |
| @storybook/addon-links | ^8.4.4 | ^8.4.7 |
| @storybook/blocks | ^8.4.4 | ^8.4.7 |
| @storybook/manager-api | ^8.4.4 | ^8.4.7 |
| @storybook/test | ^8.4.4 | ^8.4.7 |
| @storybook/theming | ^8.4.4 | ^8.4.7 |
| @storybook/web-components | ^8.4.4 | ^8.4.7 |
| @storybook/web-components-vite | ^8.4.4 | ^8.4.7 |
| eslint | ^9.15.0 | ^9.17.0 |
| prettier | ^3.3.3 | ^3.4.2 |
| sass | ^1.81.0 | ^1.83.0 |
| storybook | ^8.4.4 | ^8.4.7 |

## Testing and review

1. Confirm there are no installation errors
2. Confirm there are no build errors
3. Confirm there are no test errors
4. Prettier scripts run without error
5. Storybook builds without error
6. Run `npm start` and confirm the stroybook previews display correctly.
